### ISSUE TITLE
Merged Spectra Exported to mzML Doesn't Save MS2 Precursor Information (Reimporting to MZMine)

### ISSUE THREAD

**Issue by zglong1** [mzmine/mzmine#1608]
### Basic information

* My operating system and version: Windows 10
* My MZmine version: 3.9.0

### What happened

Two mzML files (one with MS1 data, one with MS1 and MS2 data (or just MS2 data)) were imported to mzMine. Scans were verified for each file (ie. MS1 file had MS1 scans as expected, MS2 file had MS2 scans as expected, with precursor information). The two files were merged using the Raw Data Methods -> File Merging -> Raw Data File Merging. The files were merged successfully (MS1 scans and MS2 scans from respective files were grouped into one new file; precursor information was maintained). The new file was exported as mzML. However, on loading this new combined file back into mzMine, the precursor information was lost (or at least not loaded), but everything else seems intact.

TLDR: You seemingly can't combine files (taking MS1 from one, MS2 from >=1 files), and export the combined file as mzML without losing precursor mass information.

A: 240108_MS.mzML = MS1 scans only
B: 240108_FDDA_Deep_1_MS2Only.mzML = MS1 and MS2 scans, source of MS2 scans
C: 240108_MS.mzML _wMS2 = merged file with MS1 from A and MS2 from B
D: 240108_MS_withMS2_Exported = exported C as mzML, renamed

Shared file included screenshots, mzML files, and standalone project.

https://www.dropbox.com/scl/fi/c9mbcbn9pwo58dgp5oztn/20240115_BugReport.zip?rlkey=gqd0c2px0dk0xfga31gm1mlnt&dl=0
